tools/update_assisted_installer_yaml.py

```diff
@@ -11,22 +12,56 @@
 args = parser.parse_args()
 
 
+def in_all_lists(el: str, lists: list[str]) -> bool:
+    for list in lists:
+        if el not in list:
+            return False
+    return True
+
+
+def find_first_common_element(lists: list[str]) -> str:
+    if len(lists) < 1:
+        raise IndexError("No lists provided. Cannot find common element")
+    for el in lists[0]:
+        if in_all_lists(el, lists[1:]):
+            return el
+    raise RuntimeError("Could not find common element")
```

Comment:
it seems more right to return ``None`` in this case:
```suggestion
    return None
```